config/README.md

```diff
@@ -15,3 +15,16 @@ Style-checks are done for each pull request and installing this cody style confi
 10. Press "OK"
 
 * Please let `.editorconfig` override the settings of IntelliJ
+
+
+# Eclipse:
+
+In case the Eclipse style needs to be adjusted for the eclipse.gradle file:
```

Comment:
I don't understand that. Maybe, the gradle script needs to be rewritten to inject an exported XML file`?

Reply:
I think I just found a simpler way. I will update the doc accordingly

Reply:
I generally found a nice and simple way.  One can directly export the file in eclipse as EPF file in properties format
The main drawback is that it's not possible to import it with gradle (argh!).
So therefore the only solution would be to read the file, remove all unnecessary lines and write that into the .prefs file (ideally once). Then we can just copy over the prefs file to the workspace.